spec/models/comment_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  describe 'DB table' do
+    it { is_expected.to have_db_column :body }
+  end
+
```

Comment:
you need to test the associations here

you also need to test the associations in the other unit tests as well

Reply:
Done!